src/lib/input/README.md

```diff
@@ -78,7 +78,25 @@ The divider (line under the `<md-input>` content) color can be changed by using
 
 You can label the `<md-input>` as you would a regular `<input>`.
 
+## Focus
 
+You can put the focus on an input component using the `focus()` method.
+
+### Example
+
+```html
+<md-input #nameInput placeholder="name"></md-input>
```

Comment:
The `#nameInput` isn't necessary in this example because you're using `@ViewChild` with the class. I'd omit it so that people don't mistakenly get the impression that a `@ViewChild` name has to match some template variable name.